docs/cartesian-product.adoc

```diff
@@ -119,6 +121,110 @@ To demonstrate with a table:
 | 6th test  | 4                 | "B"
 |===
 
+=== `@CartesianEnumSource`
+
+`@CartesianEnumSource` provides a convenient way to use `Enum` constants.
+
+[source,java]
+----
+@CartesianProductTest
+@CartesianEnumSource(ChronoUnit.class)
+void testWithEnumSource(TemporalUnit unit) {
+    assertNotNull(unit);
```

Comment:
Prefer empty test block with comment if possible (so it's consistent with the existing documentation). Otherwise prefer AssertJ (so it's consistent with other Pioneer documentation).

Reply:
Empty block test should be enough

Reply:
Changed to AssertJ